src/qt/transactiontablemodel.cpp

```diff
@@ -95,25 +95,27 @@ class TransactionTablePriv
      */
     QList<TransactionRecord> cachedWallet;
 
-    bool fQueueNotifications = false;
+    bool m_loaded = false;
+    int m_progress = 100;
```

Comment:
Why have an `int` when we are only interested in `bool` if progress < 100?

Reply:
Changed.